src/mpyl/project.py

```diff
@@ -336,6 +340,9 @@ def from_config(values: dict):
             command=TargetProperty.from_config(values.get("command", {})),
             args=TargetProperty.from_config(values.get("args", {})),
             labels=list(map(KeyValueProperty.from_config, values.get("labels", []))),
+            strategy_type=deployment_strategy.get("strategyType", "RollingUpdate"),
+            max_surge=deployment_strategy.get("maxSurge", "25%"),
+            max_unavailable=deployment_strategy.get("maxUnavailable", "25%"),
```

Comment:
Maybe combine the 3 new values into a `DeploymentStrategy` object or something? Also in the ChartBuilder? They belong together and it would keep the properties list a bit shorter.

Reply:
Addressed [here](https://github.com/Vandebron/mpyl/pull/356/commits/69e15036831fbe7c27b8f0e4f1c03a77a26522a4)